### PR TITLE
✨ Radiation Shield new designer

### PR DESCRIPTION
**Pull request by je-cook**
## Description

TODO testing

## Interface Changes

If you've had to update an interface or introduce a new interface as part of your change then let us know here.

## Checklist

I confirm that I have completed the following checks:

- [ ] Tests run locally and pass `pytest tests --reactor`
- [ ] Code quality checks run locally and pass `flake8` and `black .`
- [ ] Documentation built locally and checked `sphinx-build -W documentation/source documentation/build`

### CODE REVIEW

**Comment by je-cook** [comment]
#1351 should go in before this

**Review by hsaunders1904 — CHANGES_REQUESTED** [review]
Looks like some debugging code has been leftover - otherwise looks good to me

File: bluemira/builders/radiation_shield.py [hunk 1]
```diff
@@ -22,145 +22,130 @@
...
-        cryo_vv_outer = full_cryo_vv.boundary[0]
-        rs_inner = offset_wire(cryo_vv_outer, self.params.g_cr_rs)
-        rs_outer = offset_wire(rs_inner, self.params.tk_rs)
-
-        rs_full = BluemiraFace([rs_outer, rs_inner])
+        cryo_vv_rot = self.cryo_vv.deepcopy()
+        cryo_vv_rot.rotate(base=(0, 0, 0), direction=(0, 0, 1), degree=180)
+        from bluemira.display.displayer import show_cad
+
+        show_cad([self.cryo_vv, cryo_vv_rot])
```
[the remark this answers is not in the document]
Comment (je-cook) [reply bluemira/builders/radiation_shield.py:98]: Oops yeah!

File: tests/builders/test_radiation_shield.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    def test_components_and_segments(self):
+        builder = RadiationShieldBuilder(self.params, {}, self.cryo_koz)
+        rad_shield = builder.build()
+
+        assert rad_shield.component().get_component("xz")
+        assert rad_shield.component().get_component("xy")
+
+        xyz = rad_shield.component().get_component("xyz")
+        assert xyz
+        assert len(xyz.leaves) == self.params["n_TF"]["value"]
```
Comment (hsaunders1904) [inline tests/builders/test_radiation_shield.py:47]: Could we add a `show_cad` call on the `xyz` component?

**Review by hsaunders1904 — APPROVED** [review]
One suggestion for something to add to the test, but otherwise LGTM

**Pull request merged**